PVTPackage/source/MultiphaseSystem/MultiphaseSystem.cpp

```diff
@@ -157,25 +151,25 @@ TableReader::Properties TableReader::buildTables( const std::vector< pvt::PHASE_
     pvt::PHASE_TYPE const & phase = phases[i];
 
     std::vector< std::vector< double > > const & phaseTable = phaseTables[i];
-    double const & surfaceDensity = surfaceDensities[i];
+    double const & surfaceMassDensity = surfaceMassDensities[i];
     double const & molarWeight = molarWeights[i];
 
     switch( phase )
     {
       case pvt::PHASE_TYPE::OIL:
         result.oilTable = phaseTable;
-        result.oilSurfaceMassDensity = surfaceDensity;
+        result.oilSurfaceMassDensity = surfaceMassDensity;
         result.oilSurfaceMolecularWeight = molarWeight;
         break;
       case pvt::PHASE_TYPE::GAS:
         result.gasTable = phaseTable;
-        result.gasSurfaceMassDensity = surfaceDensity;
+        result.gasSurfaceMassDensity = surfaceMassDensity;
         result.gasSurfaceMolecularWeight = molarWeight;
         break;
       case pvt::PHASE_TYPE::LIQUID_WATER_RICH:
         ASSERT( phaseTable.size() == 1, "Too many lines in water properties table" );
         result.waterTable = phaseTable[0];
-        result.waterSurfaceMassDensity = surfaceDensity;
+        result.waterSurfaceMassDensity = surfaceMassDensity;
         result.waterSurfaceMolecularWeight = molarWeight;
```

Comment:
Another nitpicking comment about naming for a future PR: it noticed that `molarWeight` and `molecularWeight` are used interchangeably to designate the same thing. Maybe we could use `molarWeight` everywhere as in the input file?

Reply:
OK, I noticed it but did not know which choice was more appropriate.